documentation/src/main/asciidoc/topics/xml/persistence_file_store.xml

```diff
@@ -1,6 +1,5 @@
 <distributed-cache>
   <persistence passivation="true">
-     <file-store shared="false">
         <data path="data"/>
```

Comment:
This configuration is no longer valid as you removed the entire `file-store` element. I assume you meant to just remove the `shared` attribute?

Reply:
Returned to file. Are the .yaml and .java changes OK?

Reply:
Yes